.goreleaser.yml

```diff
@@ -0,0 +1,53 @@
+env:
+  - GO111MODULE=on
+  - CGO_ENABLED=1
+
+builds:
+  - id: regen-darwin-amd64
+    binary: regen
+    main: ./app/regen
+    goarch:
+      - amd64
+    goos:
+      - darwin
+    env:
+      - CC=/home/runner/work/osxcross/target/bin/o64-clang
+      - CXX=/home/runner/work/osxcross/target/bin/o64-clang++
+    flags:
+      - -tags="netgo ledger"
+      - -trimpath
+      - -mod=readonly
+    ldflags:
+      - -s -w -X github.com/cosmos/cosmos-sdk/version.Name=regen -X github.com/cosmos/cosmos-sdk/version.AppName=regen -X github.com/cosmos/cosmos-sdk/version.Version={{.Version}} -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}} -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger"
+  - id: regen-linux-amd64
+    binary: regen
+    main: ./app/regen
+    goarch:
+      - amd64
+    goos:
+      - linux
+    flags:
+      - -tags="netgo ledger"
+      - -trimpath
+      - -mod=readonly
+    ldflags:
+      - -s -w -X github.com/cosmos/cosmos-sdk/version.Name=regen -X github.com/cosmos/cosmos-sdk/version.AppName=regen -X github.com/cosmos/cosmos-sdk/version.Version={{.Version}} -X github.com/cosmos/cosmos-sdk/version.Commit={{.Commit}} -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger"
+archives:
+  - id: regen
+    builds:
+      - regen-darwin-amd64
+      - regen-linux-amd64
```

Comment:
Can we make separate builds & archives for experimental vs stable?

It should be as simple as creating 2 new `build` sections above with the `experimental` tag (in addiiton `tags="netgo ledger"`), and then referencing htem here in the archives section.

Reply:
@clevinson this might confuse people if we have two binaries for each release. May be we can fix this by,
- use odd number versioning for expermental tags (v1.1.x, v1.3.x, v3.1.x etc)
- use even number versioning for stable releases (v1.0.x, v1.2.x, etc)

Reply:
That looks odd. May be we can just follow -experimental for the build name and have two binaries as you suggested

Reply:
We cannot build separate builds & archieves for experimental & stable, since both of them uses the same `regen` binary for `linux_amd64` and `darwin_amd64`